### PR TITLE
Refactor the README

### PR DESCRIPTION
**Pull request by lornajane**
A revisit of the README to try to get the user onboarded more smoothly, and to add some additional resources that they might find useful.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -11,88 +11,88 @@ more information about the backend service.
...
-           @@@@        '@@@@@@@@@@@@@@@@@@@.        @@@@
-           @@@  @@@@@@+  @@@@@@@@@@@@@@@@@  @@@@@@;  @@@
-           @@  @@@@@@@@@  @@@@@@@@@@@@@@@ `@@@@@@@@@  @+
-              @@@@@@@@@@@ :@@@@@@@@@@@@@  @@@@@@@@@@@ '
-             `@@@@@@@@@@@       ```      ,@@@@@@@@@@@
-             `@@@@@@   '@                :@:   @@@@@@
-              @@@@@:                           @@@@@@
-               @@@@@                           @@@@@
-                @@@@#                         @@@@'
-
```
Comment (rominf) [inline README.rst:46]: I agree that this crab doesn't add readability. But I'll miss him. How about inserting a PNG/SVG image?

File: README.rst [hunk 2]
```diff
@@ -11,88 +11,88 @@ more information about the backend service.
...
+-----------------
+
+Pypi installation is the recommended route for most users::
 
   $ python3 -m pip install aiven-client
 
-From PyPI (Windows)::
 
-  c:\> python3 -m pip install aiven-client
+Build an rpm Package
```
Comment (rominf) [inline README.rst:43]: I'm not 100% sure but it looks like RPM should be written capitalized, so `rpm -> RPM` (here and in all other places).

File: README.rst [hunk 3]
```diff
@@ -11,88 +11,88 @@ more information about the backend service.
...
+
+.. _help-command:
+.. _basic-usage:
+
+Usage
+=====
+
+Some handy hints that work with all commands:
+
+*  The ``avn help`` command shows all commands and can _search_ for a command,
```
Comment (rominf) [inline README.rst:79]: GitHub rendres it like this:
image

You probably wanted `*search*` (*search*).

**Review by rominf — CHANGES_REQUESTED** [review]
Since you're refactoring the README, please fix the typo too: `cab be changed -> can be changed`.

Also: `<projectname> -> <project_name>` (to match the style).

Otherwise LGTM.

File: README.rst (hunk 1 above)
Comment (lornajane) [reply README.rst:46]: He can't stay here but I'd like to look at how to add `avn crab` as a sort of easter egg to still draw him when we want to see him

File: README.rst (hunk 3 above)
Comment (lornajane) [reply README.rst]: Good catch, fixed

File: README.rst (hunk 2 above)
Comment (lornajane) [reply README.rst]: Thanks, fixed it!

**Review by rominf — APPROVED** [review]

**Pull request merged**